pipeline/transformer/transformer_sierra/src/test/scala/weco/pipeline/transformer/sierra/transformers/SierraHoldingsEnumerationTest.scala

```diff
@@ -918,6 +918,17 @@ class SierraHoldingsEnumerationTest
     }
   }
 
+  it("finds a human-written description in field tag h") {
```

Comment:
For completeness, I would add a test containing both human and automatic enumerations.

I think the suite would currently pass if the code were changed to first try fetching one, then fetching the other if the first comes out empty.

Reply:
Good suggestion, will add.